priam/src/main/java/com/netflix/priam/config/PriamConfiguration.java

```diff
@@ -722,7 +722,7 @@ public String getMergedConfigurationCronExpression() {
     }
 
     @Override
-    public int getForgottenFileGracePeriodDaysForCompaction() {
+    public int getGracePeriodDaysForCompaction() {
```

Comment:
Do you want to change `PRIAM_PRE + ".forgottenFileGracePeriodDaysForCompaction` also to `gracePeriodDaysForCompaction`